kale/pipeline/mida_trainer.py

```diff
@@ -0,0 +1,200 @@
+# =============================================================================
+# Author: Shuo Zhou, shuo.zhou@sheffield.ac.uk
+# =============================================================================
+
+"""Implementation of MIDA->Feature Selection->Linear SVM/LogisticRegression Pipeline
```

Comment:
Does it makes sense that whether it can be generalized to something like `UDAGridSearchCV` or `DAGridSearchCV`?  where we note that `DA` and `UDA` imply domain adaptation and unsupervised domain adaptation specifically.

Reply:
Might be better to keep consistency with other pipelines in Pykale